FindmallocMC.cmake

```diff
@@ -96,8 +93,14 @@ if(mallocMC_ROOT_DIR)
     string(REGEX MATCH "([0-9]+)" mallocMC_VERSION_PATCH
                                 ${mallocMC_VERSION_PATCH_HPP})
 
+    # mallocMC variables ########################################################
+    #
+    set(mallocMC_REQUIRED_VARS_LIST mallocMC_ROOT_DIR mallocMC_INCLUDE_DIRS)
```

Comment:
an other regression: this needs to be placed before the `if(mallocMC_ROOT_DIR)` else it will fail with an `CMake Error at thirdParty/cmake-modules/FindmallocMC.cmake:154 (list):
  list sub-command REMOVE_DUPLICATES requires list to be present.`

I will send a pull ;)